cli.py

```diff
@@ -198,32 +198,8 @@ def pre_process(help_page: str) -> str:
     # The help page has a lot of components
     # We need to sample the `option  description` lines, which starts
     # with white spaces.
-    re_space = re.compile(r"(^\s+)")  # Regex to capture starting white spaces.
-
-    # Split the lines and iterate
-    page_splits = help_page.split("\n")
-    num_of_lines = len(page_splits)
-    for idx, line in enumerate(page_splits):
-        # Capture the starting white spaces of the line
-        white_space = re_space.findall(line)
-        num_white_space = len(white_space[0]) if white_space else 0
-        if num_white_space > 2:
-            # Can be either
-            # - wrapped description
-            # - new description
-            if num_of_lines >= idx + 1 and page_splits[idx + 1] == "":
-                # wrapped description
-                desc_1 = page_splits[idx - 1]
-                desc_2 = page_splits.pop(idx).strip()
-                page_splits[idx - 1] = desc_1 + " " + desc_2
-                # Remove the empty line
-                page_splits.pop(idx)
-            else:
-                # new description
-                option = page_splits[idx - 1]
-                description = page_splits.pop(idx).strip()
-                page_splits[idx - 1] = option + "   " + description
-    return "\n".join(page_splits)
+    re_line_indent = re.compile(r"\n {3,}", )
+    return re_line_indent.sub('  ', help_page)
```

Comment:
The whole point of these lines was to insert `'  '` (two spaces) between commands and their descriptions, while inserting `' '` (one space) between line-separated parts of descriptions. But doing this requires figuring out context about what came before and what's coming next, and it was easy to get wrong.

This solution is inelegant, but seems to work consistently. I just insert two spaces on all line breaks with large indentations, leaving those with smaller indentations alone. That means there's often more than one two-space break on a line, but it's OK: the first one is always the break between the option and the description, and the rest don't matter. So I just split on the first one, and then [collapse the rest to single spaces](https://github.com/wandb/docugen/pull/24/files#diff-7648f5f4cce4bf01d02de43f8c5f6f2130b787baece0cf6338210496d8bdfcdcR153).